projects/ngx-matomo/package.json

```diff
@@ -16,14 +16,14 @@
     "url": "https://github.com/Arnaud73/ngx-matomo.git"
   },
   "peerDependencies": {
-    "@angular/common": ">=12.0.0 <13.0.0",
```

Comment:
actually it is the 2 only mandatory lines for this MR, though I think upgrading the global package.json is also useful ?